src/main/java/com/currencyexchange/controller/CurrencyController.java

```diff
@@ -27,6 +35,19 @@ public class CurrencyController {
    * @return a {@link ResponseEntity} containing a list of currency names
    *     with an HTTP status code of 200 (OK)
    */
+  @Operation(
+      summary = "Get all available currencies",
+      description = "Retrieves a list of all available currencies from the database."
+  )
+  @ApiResponses(value = {
```

Comment:
Looks like @ApiResponses should be included into the @ApiResponses
@ApiResponses(responses = @ApiResponses())